### PR TITLE
Dx 1043

### PR DESCRIPTION
**Pull request by ogzhanolguncu**
This PR allows us to initialize chat models with type-safe functions.
Deletes togetherAi models
Some of the tests are commented for future reason they will be fixed in the following PR.

### CODE REVIEW

File: src/client/llm/providers.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+const upstash = (): {
+  owner: "upstash";
+  baseUrl: "https://qstash.upstash.io";
+  llmToken: string;
+} => {
+  if (!process.env.QSTASH_TOKEN) throw new Error("QSTASH_TOKEN cannot be empty or undefined!");
+  return {
+    owner: "upstash",
+    baseUrl: "https://qstash.upstash.io",
+    llmToken: process.env.QSTASH_TOKEN,
```
Comment (CahidArda) [inline src/client/llm/providers.ts]: not using the llmToken parameter

**Review by CahidArda — COMMENTED** [review]
line 115 in README refers to Together ai, which is removed I guess

File: src/client/llm/chat.test.ts [hunk 2]
```diff
@@ -110,61 +111,61 @@ describe("Test Qstash chat", () => {
...
-    expect(result.messageId).toBeTruthy();
-  });
+  // test("should publish with llm api", async () => {
+  //   const result = await client.publishJSON({
+  //     api: "llm",
+  //     body: {
+  //       model: "meta-llama/Meta-Llama-3-8B-Instruct",
+  //       messages: [
+  //         {
+  //           role: "user",
```
[the remark this answers is not in the document]
Comment (ogzhanolguncu) [reply src/client/llm/chat.test.ts:121]: Yes. i'll be fixed in following pr

File: src/client/llm/providers.ts (hunk 1 above)
Comment (ogzhanolguncu) [reply src/client/llm/providers.ts]: this will be wiped in the following pr.

**Pull request merged**